tools/refract-filter.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+
+# This is just a
```

Comment:
Oh I see, the comment, I moved the stuff to help instead of a comment but left few words there.